src/test/java/org/broadinstitute/hellbender/utils/mcmc/GibbsSamplerCopyRatioUnitTest.java

```diff
@@ -0,0 +1,412 @@
+package org.broadinstitute.hellbender.utils.mcmc;
```

Comment:
I suppose this is going to become the `CopyRatioModelIntegrationTest` some time soon?